### PR TITLE
#2024 - Changed NOC to 5 digits

### PR DESCRIPTION
**Pull request by andrewsignori-aot**
- Update the Education Program NOC code to accept 5 digits.
- No DB updates are required for the existing ones as per agreed by @JasonCTang.

### CODE REVIEW

**Review by sh16011993 — APPROVED** [review]
👍

**Review by guru-aot — APPROVED** [review]
LGTM

File: sources/packages/forms/src/form-definitions/educationprogram.json [hunk 1]
```diff
@@ -961,8 +962,8 @@
                         "maxLength": "",
                         "minWords": "",
                         "maxWords": "",
-                        "pattern": "[0-9]{4}",
-                        "customMessage": "Incorrect Format'",
+                        "pattern": "[0-9]{5}",
+                        "customMessage": "Incorrect Format",
```
Comment (ann-aot) [inline sources/packages/forms/src/form-definitions/educationprogram.json:966]: 👍

**Review by ann-aot — APPROVED** [review]
👍

**Review by dheepak-aot — APPROVED** [review]
👍

**Review by andrepestana-aot — COMMENTED** [review]
Sorry for the late review. looks good!

**Pull request merged**